plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -42,6 +41,8 @@ import (
 	"k8s.io/kubernetes/plugin/pkg/scheduler/schedulercache"
 	schedutil "k8s.io/kubernetes/plugin/pkg/scheduler/util"
 	"k8s.io/metrics/pkg/client/clientset_generated/clientset"
+
+	"github.com/golang/glog"
```

Comment:
Sure, but it's not consistent in the codebase.  e.g. why spend cycles on this if we don't have a standard.